diff --git a/bundles/chunk-7CTS3DE5.js b/bundles/chunk-7CTS3DE5.js
index c2dac3e174206f2e1e701cfc0a45cb9a12754d15..517e0a50292f96676dea638cf44014dfbf853a31 100755
--- a/bundles/chunk-7CTS3DE5.js
+++ b/bundles/chunk-7CTS3DE5.js
@@ -4189,7 +4189,7 @@ var NgCompiler = class {
     let aliasingHost = null;
     if (this.adapter.unifiedModulesHost === null || !this.options["_useHostForImportGeneration"] && !this.options["_useHostForImportAndAliasGeneration"]) {
       let localImportStrategy;
-      if (this.options.rootDir !== void 0 || this.options.rootDirs !== void 0 && this.options.rootDirs.length > 0) {
+      if (this.options.rootDirs !== void 0 && this.options.rootDirs.length > 0) {
         localImportStrategy = new LogicalProjectStrategy(reflector, new LogicalFileSystem([...this.adapter.rootDirs], this.adapter));
       } else {
         localImportStrategy = new RelativePathStrategy(reflector);
